Code/GraphMol/FileParsers/SDWriter.cpp

```diff
@@ -114,16 +114,16 @@ void _MolToSDStream(std::ostream *dp_ostream, const ROMol &mol, int confId,
     // if use did not specify any properties, write all non computed properties
     // out to the file
     STR_VECT properties = mol.getPropList();
-    STR_VECT compLst;
-    mol.getPropIfPresent(detail::computedPropName, compLst);
-
+    STR_VECT compLst = mol.getComputedPropList();
+    
     STR_VECT_CI pi;
     for (pi = properties.begin(); pi != properties.end(); pi++) {
       // ignore any of the following properties
       if (((*pi) == detail::computedPropName) ||
-          ((*pi) == common_properties::_Name) || ((*pi) == "_MolFileInfo") ||
-          ((*pi) == "_MolFileComments") ||
-          ((*pi) == common_properties::_MolFileChiralFlag)) {
+          ((*pi) == common_properties::getPropName(common_properties::_Name)) ||
+          ((*pi) == common_properties::getPropName(common_properties::MolFileInfo)) ||
+          ((*pi) == common_properties::getPropName(common_properties::MolFileComments)) ||
+          ((*pi) == common_properties::getPropName(common_properties::_MolFileChiralFlag))) {
         continue;
```

Comment:
Yes, I think we can do that in a second pass.  I wanted to make these as similar as possible to avoid brain explodings.